Content.Server/GameTicking/Rules/NukeopsRuleSystem.cs

```diff
@@ -793,7 +797,8 @@ private void OnMindAdded(EntityUid uid, NukeOperativeComponent component, MindAd
 
             if (nukeops.TargetStation != null && !string.IsNullOrEmpty(Name(nukeops.TargetStation.Value)))
             {
-                _chatManager.DispatchServerMessage(playerSession, Loc.GetString("nukeops-welcome", ("station", nukeops.TargetStation.Value)));
+                var operation_name = _randomMetadata.GetRandomFromSegments(new List<string> {"operationPrefix", "operationSuffix"}, " ");
```

Comment:
This seems like it'll cause the operation name to change for every new nukie? Might want to store it somewhere.

Reply:
store in the nukeops rule then just tell nukies the field

Reply:
oh yeah ill try to edit that, sorry it's the first time i add something

Reply:
would you recommend i just remove that line or... ?